.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,47 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      # 👇 Cache dependencies
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        run: yarn install
+        
+      # 👇 Adds Chromatic as a step in the workflow
+      - name: Build Storybook
+        run: yarn workspace @okta/odyssey-storybook build
+
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        
+        # Chromatic GitHub Action options
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # 👇 Chromatic projectToken, refer to the manage page to obtain it.
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
are these secrets uploaded through the github web interface to our repo?

Reply:
Yep, if you go to Settings > Secrets you can put them in there. An admin has to do this (which is why I wrote these POCs on my personal account)